cmd/buildkitd/main_containerd_worker.go

```diff
@@ -328,7 +328,7 @@ func containerdWorkerInitializer(c *cli.Context, common workerInitializerOpt) ([
 			Options: opts,
 		}
 	}
-	opt, err := containerd.NewWorkerOpt(common.config.Root, cfg.Address, snapshotter, cfg.Namespace, cfg.Rootless, cfg.Labels, dns, nc, common.config.Workers.Containerd.ApparmorProfile, common.config.Workers.Containerd.SELinux, parallelismSem, common.traceSocket, runtime, ctd.WithTimeout(60*time.Second))
+	opt, err := containerd.NewWorkerOpt(common.config.Root, cfg.Address, snapshotter, cfg.Namespace, cfg.CgroupParent, cfg.Rootless, cfg.Labels, dns, nc, common.config.Workers.Containerd.ApparmorProfile, common.config.Workers.Containerd.SELinux, parallelismSem, common.traceSocket, runtime, ctd.WithTimeout(60*time.Second))
```

Comment:
This list of argument is getting really really long, and probably requires changes in Moby as well; should we implement a struct for passing options? Similar to https://github.com/moby/moby/blob/450f18d3cae6be625d671b6bacd3eefcaaa13bb9/builder/builder-next/builder.go#L79 ?

Reply:
Ah, I wanted to comment on the other file 🙈 same applies 😅

Reply:
Don't refactor this in the same PR. Future follow-up welcome.